comixed-frontend/src/app/library/components/convert-comics-settings/convert-comics-settings.component.html

```diff
@@ -24,6 +24,14 @@
                   [binary]='true'
                   formControlName='deletePages'></p-checkbox>
     </div>
+    <div class='ui-g-12'>
+      <label for='delete-pages'>Delete comic</label>
```

Comment:
This text should go into the library translation file.

Reply:
i forgot to change this.